modules/gitops-release-notes-1-13-1.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-13-1_{context}"]
+= Release notes for {gitops-title} 1.13.1
+
+{gitops-title} 1.13.1 is now available on {OCP} 4.12, 4.13, 4.14, 4.15, and 4.16.
+
+[id="errata-updates-1-13.1_{context}"]
+== Errata updates
+
+[id="rhsa-XXXX:NNNN-gitops-1-13-1-security-update-advisory_{context}"]
+=== RHSA-XXXX:NNNN - {gitops-title} 1.13.1 security update advisory
+
+Issued: 2024-07-25
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-XXXX:NNNN[RHSA-XXXX:NNNN]
```

Comment:
Going to assume getting 404 error message as the doc is not live yet, but wanted to point out just in case.

Reply:
Hi @mletalie that is true as the errata is not published yet so we have left it as is.